flutter_app/android/app/src/main/java/com/exotel/voice_sample/ExotelSDKChannel.java

```diff
@@ -394,7 +402,7 @@ public void onStatusChange() {
              */
             channel.invokeMethod("loggedInStatus",mService.getCurrentStatus().getMessage());
         });
-
+        fetchContactList();
```

Comment:
create new contract b/w flutter and native to fetch contact list.
when user go to contact tab, new fetch request will triggered and new list will be updated.